app/models/sponsor.rb

```diff
@@ -2,6 +2,8 @@ class Sponsor < ActiveRecord::Base
   include Initializer
   NEW_CITY_MENU_OPTION = '**Add New**'
   PAYMENT_PLANS = ['Monthly', 'Every Two Months', 'Every Four Months', 'Every Six Months', 'Annually', 'Other']
+  PRIORITY_COUNTRIES= %w(SA TR AE GB)
+  EXCLUDED_COUNTRYS= %w(IL BQ BV IO KY CX CC CW FK TF GG HM IM JE YT AN NF NO PN BL SH MF PM SX GS SS SJ TK TC UM AX)
```

Comment:
YES.
Or we can change `en_ar_country` to return only the English version on a missing translation.
But the missing countries are not that important:

BQ (Bonaire, Sint Eustatius and Saba) 
BV (Bouvet Island)
IO (British Indian Ocean Territory)
KY (Cayman Islands)
CX (Christmas Island)
CC (Cocos (Keeling) Islands)
CW (Curaçao)
FK (Falkland Islands (Malvinas)) 
TF (French Southern Territories)
GG (Guernsey)
HM (Heard and McDonald Islands)
IM (Isle of Man) 
JE (Jersey)
YT (Mayotte)
AN (Netherlands Antilles)
NF (Norfolk Island) 
PN (Pitcairn)
BL (Saint Barthélemy) 
SH (Saint Helena)
MF (Saint Martin) 
PM (Saint Pierre And Miquelon)
SX (Sint Maarten) 
GS (South Georgia and the South Sandwich Islands) 
SS (South Sudan)
SJ (Svalbard And Jan Mayen)
TK (Tokelau)
TC (Turks and Caicos Islands) 
UM (United States Minor Outlying Islands) 
AX (Åland Islands)